tests/test_init.py

```diff
@@ -0,0 +1,412 @@
+# import configparser
+import configparser
+from pickle import TRUE
+from unittest import mock
+
+# import script that needs to be tested
+import reportdaily as rd
+# required for fixture
+import pathlib
+import os
+# required for date
+from datetime import date
+# import namespace
+from argparse import Namespace
+# monkey patching /mocker patch
+import unittest
+from unittest.mock import MagicMock, patch
+import builtins
+# required fo tests
+import pytest
+
+
+# Global data
+STANDARD_SECTIONS = ["settings"]
+STANDARD_OPTIONS = ["name", "team", "start_year", "current_day"]
+STANDARD_CONFIG = {  # Section_name: list of key names
+    "settings": ("name", "team", "start_year", "current_day"),
```

Comment:
Be DRY ("don't repeat yourself") here:

```suggestion
    "settings": tuple(STANDARD_OPTIONS),
```

Reply:
Thanks your are right  :dancers: